packages/synapse-interface/pages/state-managed-bridge/index.tsx

```diff
@@ -174,6 +166,15 @@ const StateManagedBridge = () => {
         (quote) => quote.bridgeModuleName === 'SynapseRFQ'
       )
 
+      let quote
```

Comment:
Do we want to check `allQuotes` for not being an empty list here?

`allBridgeQuotes` doesn't throw an error, if no quotes have been found and just returns an empty list instead.

Reply:
This being said, the test build still handles the empty list case correctly, as the error is thrown here (`quote` is undefined there)

https://github.com/synapsecns/sanguine/blob/0adc35cb771bd4d59ed61e0442b3e883503c09a6/packages/synapse-interface/pages/state-managed-bridge/index.tsx#L178-L186

Reply:
Yep, makes sense, handling [here](https://github.com/synapsecns/sanguine/pull/1861/commits/4e7c89a752645705149bb48907ce057420170ab9)

Reply:
Perfect, thanks!